experimental/framework/frs-client/README.md

```diff
@@ -44,10 +65,13 @@ Using the `FrsClient` object the developer can create and get Fluid containers.
 ```javascript
 import { FrsClient } from "@fluid-experimental/frs-client";
 
-await FrsClient.createContainer( { id: "_unique-id_" }, /* schema */);
-const [container, containerServices] = await FrsClient.getContainer({ id: "_unique-id_" }, /* schema */);
+const frsClient = new FrsClient(config);
+await frsClient.createContainer( { id: "_unique-id_" }, /* schema */);
+const { fluidContainer, containerServices } = await frsClient.getContainer({ id: "_unique-id_" }, /* schema */);
```

Comment:
Is this frsResrouces change was reflected in the readme for tinylicious-client as well?

Reply:
Yep @sdeshpande3 has applied the changes there